src/nupic/algorithms/SpatialPooler.cpp

```diff
@@ -35,6 +35,12 @@
 
 #include <nupic/algorithms/SpatialPooler.hpp>
 #include <nupic/math/Math.hpp>
+
+#if defined(NTA_OS_WINDOWS)
+// winnt.h defines VOID and clashes with capnp::VOID
```

Comment:
If `capnp::VOID` is namespaced then it shouldn't be a problem. And this really seems out of place in the SpatialPooler.